light/client.go

```diff
@@ -1010,6 +1009,11 @@ func (c *Client) findNewPrimary(ctx context.Context, height int64, remove bool)
 			select {
 			case witnessResponsesC <- witnessResponse{lb, witnessIndex, err}:
 			case <-ctx.Done():
+				witnessResponsesC <- witnessResponse{
+					nil,
```

Comment:
we should still pass lb in just to be safe

Reply:
refactored, so no longer relevant